qa/workunits/rados/test_rados_tool.sh

```diff
@@ -52,18 +52,30 @@ run() {
 DNAME="`dirname $0`"
 DNAME="`readlink -f $DNAME`"
 RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
-if ! test -f $RADOS_TOOL ; then
-    RADOS_TOOL=$(which rados)
+if [ ! -f "$RADOS_TOOL" ] ; then
```

Comment:
On 7-7-2016 12:29, Kefu Chai wrote:
> In qa/workunits/rados/test_rados_tool.sh
> <https://github.com/ceph/ceph/pull/10166#discussion_r69884463>:
> 
>> @@ -52,18 +52,30 @@ run() {
>>  DNAME="`dirname $0`"
>>  DNAME="`readlink -f $DNAME`"
>>  RADOS_TOOL="`readlink -f \"$DNAME/../rados\"`"
>> -if ! test -f $RADOS_TOOL ; then
>> -    RADOS_TOOL=$(which rados)
>> +if [ ! -f "$RADOS_TOOL" ] ; then
> 
> we don't use |CEPH_TMP| AFAIK. i am trying to move intermediate stuff
> into |$TMPDIR| instead.

But then why don't we/you do that in the |detect-build-env-vars.sh|

And then start to try and propagate it into scripts as we go along.
Because my /tmp looks like a battlezone after running all tests.
Lots of the unittest could be more careful as well.